### ISSUE TITLE
Text Encoder and Training Steps question (fastDreambooth)

### ISSUE THREAD

**Issue by FollowMarcos** [TheLastBen/fast-stable-diffusion#1461]
I haven't used the colab in awhile but I've now noticed that it trains the text encoder and the then training steps differently. From what I remember if I had, for example, the training steps set to 1000 and the text encoder to 200, it would first train the text encoder to 200 and then train the remaining 800 steps as the training steps. Now it trains 200 txt encoder steps and 1000 training steps separately.

Is there a reason for this? I am asking because it's now delivering me different results. Do I now need to remove the same amount of steps I've used on the text encoder from the training steps? Text encoder 200 and training steps 800?

I might just be seeing things.

But if that
Cheers.

**Comment by TheLastBen** [comment]
The different result is due to the learning rate, the previous setttings for the learning rates is 2e-6 for both

**Comment by FollowMarcos** [comment]
So if I want to achieve the results like before I need to set it back to 2e-6 for both?
